app/views/visitor_mailer/booked.html.erb

```diff
@@ -103,5 +103,5 @@
 </p>
 
 <p>
-  <small><%= t(".visit_id") %> <%= @visit&.id %></small>
+  <small><%= t(".visit_id") %> <%= @visit.human_id %></small>
```

Comment:
I was wondering why it was there in the first place, all visits should have an `id` or `human_id`. If there is a good reason for I can put it back here as well as changing the rest of the emails